app/src/main/kotlin/com/wire/android/ui/home/newconversation/groupOptions/GroupOptionsScreen.kt

```diff
@@ -295,6 +302,30 @@ private fun GroupOptionState.AllowGuestsOptions(isChannel: Boolean, onAllowGuest
     }
 }
 
+@Composable
+private fun GroupOptionState.EnableWireCellOptions(onEnableWireCell: (Boolean) -> Unit) {
+    GroupConversationOptionsItem(
+        title = stringResource(R.string.enable_wire_cell),
+        switchState = SwitchState.Enabled(
+            value = isWireCellsEnabled ?: false,
+            isOnOffVisible = false,
+            onCheckedChange = { onEnableWireCell.invoke(it) }
+        ),
+        arrowType = ArrowType.NONE,
+        clickable = Clickable(enabled = false, onClick = {}),
+        modifier = Modifier.background(MaterialTheme.colorScheme.surface)
+    )
+
+    Text(
+        text = stringResource(R.string.enable_wire_cell_switch_description),
+        fontWeight = FontWeight.Normal,
+        color = MaterialTheme.wireColorScheme.secondaryText,
+        modifier = Modifier.padding(MaterialTheme.wireDimensions.spacing16x),
+        textAlign = TextAlign.Left,
+        fontSize = 16.sp
+    )
```

Comment:
Can you replace  all `fontWeight` and `fontSize` with `style = typography().body01` on this screen?
I checked and designs say that it should be `body01`, but probably this screen has never been updated to use only typography and not hardcode any font parameters 😅